CHANGELOG.md

```diff
@@ -6,6 +6,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [8.2.1]
+### Changed
+- Bump @metamask/auto-changelog from 3.4.2 to 3.4.3 ([#155](https://github.com/MetaMask/utils/pull/155))
+- Bump @metamask/auto-changelog from 3.3.0 to 3.4.2 ([#154](https://github.com/MetaMask/utils/pull/154))
+
```

Comment:
This is a dev dep change, so we can omit it since it doesn't impact the package

```suggestion
```

Reply:
good call ty ty